src/node/server.ts

```diff
@@ -10,7 +10,7 @@ export async function createServer(
 
   return createViteServer({
     root: config.srcDir,
-    base: config.site.base,
+    base: serverOptions.base ?? config.site.base,
```

Comment:
Yeah, this won't work, because the rest of VitePress needs it to be in site config itself. This option is to pass that to Vite only.

Reply:
Changed as your suggestion.